zml/hostbuffer.zig

```diff
@@ -251,6 +256,20 @@ pub const HostBuffer = struct {
         };
     }
 
+    pub fn awaitt(self: *HostBuffer) !*HostBuffer {
+        if (self._ready) {
```

Comment:
I'm not sure how I feel about this, it's easy to forget to call await.
Can we call it ourselves in a few places ? like format, or items ?